src/main.py

```diff
@@ -68,28 +70,33 @@ def methodology(year, lang):
 def chapter(year, chapter, lang):
     # TODO: Validate the chapter.
     # TODO: Get chapter data and pass into the template.
-    return render_template('%s/%s/chapter.html' % (lang, year), chapter=chapter)
+    return render_template('%s/%s/chapters/%s.html' % (lang, year, chapter), chapter=chapter)
```

Comment:
The `chapter` URL slug is no longer useful to the template, so it can be removed.
```suggestion
    return render_template('%s/%s/chapters/%s.html' % (lang, year, chapter))
```